templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -64,18 +64,26 @@ Metadata:
           - ESSnapshotId
           - HomeVolumeSnapshotId
           - HomeDeleteOnTermination
+          - BitbucketLicenseKey
+          - BitbucketAdminPassword
+          - BitbucketDatasetURL
       - Label:
           default: AWS Quick Start Configuration
         Parameters:
           - QSS3BucketName
           - QSS3KeyPrefix
           - ExportPrefix
-
     ParameterLabels:
       BitbucketProperties:
         default: Bitbucket properties
       BitbucketVersion:
         default: Version *
+      BitbucketLicenseKey:
+        default: License Key for Bitbucket (if you have one)
```

Comment:
Do we need `(if you have one)` there if we have also `(Optional)` qualifier for the parameter?